packages/arb-token-bridge-ui/synpress.config.ts

```diff
@@ -48,6 +53,24 @@ export default defineConfig({
       ) =>
         TestWETH9__factory.connect(tokenAddress, testWallet.connect(provider))
 
+      // Deploy test ERC-20 token
+      const erc20Contract = new TestERC20__factory().connect(
+        wallet.connect(ethProvider)
+      )
+      const l1Erc20Token = await erc20Contract.deploy()
+      await l1Erc20Token.deployed()
+
+      const l2Network = await getL2Network(wallet.connect(arbProvider))
+      const erc20Bridger = new Erc20Bridger(l2Network)
+
+      // Deploy to L2
+      await erc20Bridger.deposit({
```

Comment:
Is there a reason this code isn't in `on('before:run'`? Will it work properly?

Reply:
I run it here because cypress env vars need to be set from the results we get. We could move some of these calls to `before:run` but I found it to be less organised since the flow is mixed up and may be harder to understand.